doc/mf6io/prt/prt.tex

```diff
@@ -21,6 +21,48 @@ \subsection{Specifying Cell Face Flows using IFLOWFACE}
 \subsection{Particle Mass Budget}
 A summary of all inflow (sources) and outflow (sinks) of particle mass is called a mass budget.  The particle mass budget is printed to the PRT Model Listing File for selected time steps.  In the current implementation, each particle is assigned unit mass, and the numerical value of the flow can be interpreted as particles per time.
 
+\subsection{Vertical Tracking}
+
+When a particle is in the flow field, vertical motion can be solved in the same way as lateral motion. Special handling is necessary above the water table.
+
+A dry cell is either an inactive cell or an active but dry cell, as can occur with the Newton formulation.
+
+Normally, an inactive cell might be dry or explicitly disabled (idomain).  With Newton, dry cells remain active.
+
+Of the flow model, PRT knows only what the FMI or exchange tells it. This includes heads, flows and the active grid region, but not whether Newton is on, where boundary packages are, or even which boundary packages are present.
+
+Tracking and termination decisions must be made, therefore, on the basis of available information like whether a cell is active, whether the cell is dry, and whether the particle is dry (above the water table).
+
+Release-time and tracking-time considerations are described (and implemented) separately.
+
+\subsubsection{Release}
+
+At release time, PRT decides whether to release each particle or to terminate it unreleased.
+
+If the release cell is active, the particle will be released at the specified coordinates.
+
+If the release cell is inactive, behavior is determined by the DRAPE option. If the DRAPE option is enabled, the particle will be released from the top-most active cell beneath it, if any. If there is no active cell underneath the particle in any layer, or if DRAPE is not enabled, the particle will terminate unreleased (with status code 8).
+
+Since under the Newton formulation dry cells can remain active, the DRAPE option has no effect when Newton is on (assuming particles are not released into disabled grid regions). Vertical tracking behavior with Newton can be configured with tracking-time settings.
+
+\subsubsection{Tracking}
+
+A particle might find itself above the water table for one of two reasons: it was released above the water table, or the water table has receded.
+
+With the Newton formulation, particles can be released into dry-but-active cells.
+
+Particle trajectories are solved over the same time discretization used by the flow model. A particle may be immersed in the flow field in one time step, and find that the water table has dropped below it in the next.
```

Comment:
not sure we need the first sentence in this paragrpah

Reply:
removed it